### DISCUSSION TITLE
O./C. Longitude definition

### DISCUSSION THREAD

**Discussion by peacetemple** [Stellarium/stellarium#3168] [Q&A] [answered]
What does the O. and C. stand for?

**Comment by alex-w** [comment, marked as answer]
From the tooltip: Opposition/conjunction longitude line - the line of ecliptic longitude which passes through both ecliptic poles, the Sun and opposition point.

**Reply by peacetemple** [reply]
much appreciated

**Comment by gzotti** [comment]
Also User Guide, section 18.2.4.
